src/background/service/userWallet.ts

```diff
@@ -767,20 +800,18 @@ class UserWallet {
 
     // Get the current public key
     const pubKey = this.store.currentPubkey;
+    const matchingAccount = await this.getCurrentAccount();
```

Comment:
There is no need to lookup keyindexer everytime to find the account. The account should be cached and load directly from local